_posts/2020-07-22-internal-projects.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Internal Projects"
```

Comment:
hm, we should make the title a bit more catchy 

Maybe something like "The importance of working on internal projects" or "How we work on internal projects"